hook/run.go

```diff
@@ -0,0 +1,71 @@
+package hook
+
+import (
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+	// +kubebuilder:scaffold:imports
+)
+
+var (
+	scheme   = runtime.NewScheme()
+	setupLog = ctrl.Log.WithName("setup")
+)
+
+func init() {
+	if err := clientgoscheme.AddToScheme(scheme); err != nil {
+		panic(err)
+	}
+
+	// +kubebuilder:scaffold:scheme
+}
+
+// Run runs the webhook server.
+func Run(cfg *rest.Config, webhookHost string, webhookPort int, metricsAddr, certDir string) error {
+	ctrl.SetLogger(zap.Logger(false))
```

Comment:
It is difficult to read a non-development log.
I want to be able to set a flag like topolvm-node.
https://github.com/cybozu-go/topolvm/blob/c015ea67a8f789f0eb53c498ae92cb55b9311192/pkg/topolvm-node/cmd/root.go#L75